lib/dice/src/handoff.rs

```diff
@@ -0,0 +1,90 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::{AliasCert, AliasOkm, DeviceIdCert};
+use hubpack::SerializedSize;
+use lpc55_pac::syscon::RegisterBlock;
+use serde::{Deserialize, Serialize};
+
+// This memory is the USB peripheral SRAM that's 0x4000 bytes long. Changes
+// to this address must be coordinated with the [dice_*] tables in
+// chips/lpc55/chip.toml
+// TODO: get from app.toml -> chip.toml at build time
+const MEM_START: usize = 0x4010_0000;
+const ALIAS_START: usize = MEM_START;
+
+/// The Handoff type is a thin wrapper over the memory region used to transfer
+/// DICE artifacts (seeds & certs) from stage0 to hubris tasks. It is intended
+/// for use by stage0 to write these artifacts to memory where they will later
+/// be read out by a hubris task.
+pub struct Handoff<'a>(&'a RegisterBlock);
```

Comment:
It could. The use case described in https://rfd.shared.oxide.computer/rfd/0282#_ram is pretty limited though. I'm trying to keep things simple and explicit. I'm not against a more generic interface but deferring till there's a use case driving it seems prudent.